media-libs/opensubdiv/opensubdiv-3.4.0.recipe

```diff
@@ -7,26 +7,20 @@ surface matches Pixar's Renderman to numerical precision."
 HOMEPAGE="http://graphics.pixar.com/opensubdiv/"
 COPYRIGHT="2013-2018 Pixar"
 LICENSE="Apache v2"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="https://github.com/PixarAnimationStudios/OpenSubdiv/archive/v${portVersion//\./_}.tar.gz"
 CHECKSUM_SHA256="d932b292f83371c7518960b2135c7a5b931efb43cdd8720e0b27268a698973e4"
 SOURCE_FILENAME="opensubdiv-$portVersion.tar.gz"
 SOURCE_DIR="OpenSubdiv-${portVersion//\./_}"
-PATCHES="opensubdiv-$portVersion.patchset"
 
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
-SECONDARY_ARCHITECTURES="?x86"
+SECONDARY_ARCHITECTURES="x86"
 
 libVersion="$portVersion"
 libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
 
 PROVIDES="
 	opensubdiv$secondaryArchSuffix = $portVersion
-	cmd:far_perf = $portVersion
-	cmd:far_regression = $portVersion
-	cmd:hbr_baseline = $portVersion
-	cmd:hbr_regression = $portVersion
-	cmd:stringify = $portVersion
```

Comment:
Why remove commands?

Reply:
They are provided in the _tools package (and not by the base package)

Reply:
They moved to _tool subpackage, but i can't seems to find the PROVIDES section for the subpackage.

Reply:
Here? https://github.com/haikuports/haikuports/blob/ee0725aae337f986e134c26dea661c91d3cd4708/media-libs/opensubdiv/opensubdiv-3.4.0.recipe#L41

Reply:
I see, but why can't we see that in the diff? Or what the heck is going on here?

Reply:
You already added it to the previous version, so no need to change anything there :)